lib/configure.js

```diff
@@ -48,6 +48,7 @@ function configure (gyp, argv, callback) {
         }
       } else {
         log.verbose('`which` succeeded', python, execPath)
+        python = execPath
```

Comment:
the `which` module is more similar to windows PATH resolution semantics then the following require('child_process').execFile in line 85.
`execFile` fails to resolve *.bat and *.cmd files from the PATH, but will run them if explicitly specified to.